market/connector/twitter/demo/twitterDemoTest/src/com/twitter/test/json/tweets.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": [
+    {
+      "id": "1338916725543342081",
+      "text": "Hat mal wieder viel Spass gemacht! Vielen Dank an alle Teilnehmer und die Kollegen von @blue_prism und @axonivy https://t.co/v7a2ifO5ae"
```

Comment:
is this a real tweet? 🤣

Reply:
![twitterToday](https://user-images.githubusercontent.com/15085693/102618189-a493f280-413a-11eb-80da-033bb3b26a9c.png)

Of course, I'm always trying to involve real world data in my tests :D

Reply:
Haha nice 😀